bukkit/src/main/java/io/github/wysohn/triggerreactor/bukkit/main/BukkitEventRegistry.java

```diff
@@ -98,17 +100,34 @@ public Class<?> getEvent(String eventStr) throws ClassNotFoundException {
         return event;
     }
 
+    private EventPriority getPriority(int priority) {
```

Comment:
Maybe we should change the visibility modifier `private` → `public`, but just my opinion

Reply:
> Maybe we should change the visibility modifier `private` → `public`, but just my opinion

Sounds legit, but this is merely to map the priority value to the `EventPriority`, so there wouldn't be much usecase for it other than the BukkitEventRegistry class itself

Lets comeback to it and fix it if we really need it

Reply:
Ahh, that makes sense a lot.